duckduckgo/iOS
# Add pixels to measure set as default

Task/Issue URL: https://app.asana.com/0/0/1205353165912634/f
Tech Design URL:
CC:

**Description**:

Adds pixels to measure clicks on the onboarding buttons and wether we've been set as default using opening a URL within the last 7 days as a proxy.

**Steps to test this PR**:
1. Test that clicking on set as default screen buttons fire the correct pixel 
2. When performing a search the app or launching the app, it should fire a daily pixel with `default_browser` parameter set to true or false
3. To set it to true set the app as default then launch a URL (e.g. from a reminder) 
4. After confirming the DailyPixel only fires once per day hack the code in `SetAsDefaultStatistics.swift` so that it's a regular pixel and you will be able to observe the pixel and its parameter
5. After setting default to `true`, update the Date in the `isDefault` check to `Date.distantFuture` and ensure that the pixel sends with `default_browser` as `false`



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
